contents/product-engineers/activation-metrics.mdx

```diff
@@ -0,0 +1,258 @@
+---
+date: 2024-11-05
+title: How we found our activation metric (and how you can too)
+author:
+  - raquel-m-smith
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-marketing/marketing-hog.jpg
+featuredImageType: full
+category: Engineering
+tags:
+  - Product analytics
+  - Product engineers
+  - Growth
+  - Inside PostHog
+crosspost:
+  - Blog
+---
+
+Activation AKA your product's "ah-ha" moment and the second A in the [AARRR pirate funnel framework](/product-engineers/aarrr-pirate-funnel). It's the thing that lets you know if your users are going to find success with your product.
+
+In a word, it's _important_, but it's also really difficult to define.
+
+We have 8+ products and users get activated into _each_ separately. Each also has 10+ events that we could use as an activation metric. 
+
+This means we spent a lot of time figuring out how to _figure out_ what our activation metrics are. Lucky for you, we did this all inside of PostHog, so we figured we'd share it with you, so you can do the same!
```

Comment:
- Not sure if Im a fan of the +, but feel free to keep
- "figuring out how to figure out" reads slighly awkward (I get what you're trying to do though)
- Not sure I like the  sentence"Lucky for you, we did this all inside of PostHog, so we figured we'd share it with you, so you can do the same!". Feels a bit condescending. I don't have an alternative, though

```suggestion
We have more than 8 products and each also has more than 10 events that we could use as for activation, so we've spent a lot of time thinking about it. 

Lucky for you, we did this all inside of PostHog, so we figured we'd share it with you, so you can do the same!

```